controllers/users.js

```diff
@@ -1,58 +1,144 @@
-import models from '../models/';
 import jwt from 'jsonwebtoken';
 import dotenv from 'dotenv';
-const User = models.User;
+import crypto from 'crypto';
+import sgMail from '@sendgrid/mail';
+import userModels from '../models/';
+const { User } = userModels;
 
 dotenv.config();
 
 class Users {
-    static async createUser(req, res) {
-        const {id, displayName, emails, provider} = req.user;
```

Comment:
@fridolinho before you remove code please cross check if you really understand what it does or ask a team member. for example from line 10 to 54. that code is connecting Facebook and google passport strategy to connect social logins.
My advice would be for you to put it back, please.

Reply:
Actually I changed only the indentation, but github shows it as removed but codes are there